### PR TITLE
Added 3 new quotes

### PR DESCRIPTION
**Pull request by xiefiona**
added quote by Gaston Bachelard, Edmund De Waal, and Ludwig Wittgenstein

#13

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -204,11 +204,23 @@
...
-     "author": "Gaston Bachelard"
+    "sentence": "One should not aim at being possible to understand but at being impossible to misunderstand.",
+    "author": "Marcus Fabius Quintilian"
+ },
+ {
+    "sentence": "With languages, you are at home anywhere.",
+    "author": "Edmund De Waal"
+ },
+ {
+    "sentence": "The limits of my language are the limits of my world.",
```
Comment (jayehernandez) [inline server/data/quotes.json:223]: This is a duplicate, please see lines `110-114`. Hope you can remove it before I merge :)

Comment (xiefiona) [reply server/data/quotes.json]: yes I changed it. Let me know if it was deleted :)

Comment (jayehernandez) [reply server/data/quotes.json]: Great thanks!

**Comment by jayehernandez** [comment]
Thanks for contributing @xiefiona! 🎉

@all-contributors please add @xiefiona for content

**Pull request merged**